.github/dependabot.yml

```diff
@@ -22,6 +22,10 @@ updates:
             - ">= 27"
 
       # Is Vue 3 only
+      -   dependency-name: vue
```

Comment:
The above comment refers to vue-loader I guess, so it should be kept there. Also, the indentation seems off (?).

Reply:
Technically the portal-vue dependency falls under this too but you are right, I have added a comment that we can only enable vue3 dependabot PRs if we are ready to migrate to vue 3

Also the indentation of this file was a little off and I have fixed this too.

Both done here: https://github.com/demos-europe/demosplan-core/pull/775/commits/856d2fb78da5911426c031ccaf2eef82e5b7e294